packages/grafana-ui/src/components/TimePicker/TimePicker.tsx

```diff
@@ -152,7 +167,13 @@ export class TimePicker extends PureComponent<Props, State> {
             </button>
           )}
           <ButtonSelect
-            className="time-picker-button-select"
+            className={`time-picker-button-select ${
+              rightButton
+                ? rightButton.active
+                  ? 'btn--radius-right-0 active-time-picker time-picker--border-right-0'
```

Comment:
I suggest migrating this component to Emotion. Introducing new class names, especially class names that are such specific is not sth we wanna do :)

Reply:
👍 great point, I will use Emotion and therefore no new classnames will be introduced.